code/modules/projectiles/gun.dm

```diff
@@ -201,7 +201,7 @@
 
 /obj/item/weapon/gun/proc/process_fire(atom/target as mob|obj|turf, mob/living/user as mob|obj, message = 1, params, zone_override)
 	add_fingerprint(user)
-
+	add_logs(usr, target, "shot at", admin=0)
```

Comment:
Il be honest, I litteraly copy pasted my earlier changes.

Thank you for spotting that.